couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -201,8 +201,7 @@ def collect_version(self, data):
 
         # Next, get all the nodes
         if nodes is not None:
-            for node in nodes:
-                version = node['version']
+            version = nodes[0]['version']
 
         self.set_metadata('version', version)
```

Comment:
```suggestion
    if nodes is not None:
         version = nodes[0]['version']
        self.set_metadata('version', version)
```
Let's only set metadata if version is found. We could also remove `version = ""` as well if we only use version when it exists. Otherwise, we'd set version to empty string.